requirements/pip.txt

```diff
@@ -0,0 +1,6 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    make upgrade
+#
```

Comment:
`pip.in` is empty, so this one is essentially also; don't you want a few things like `pip`, `setuptools`, and `wheel` in there?

Reply:
Updated the `pip.in` file with `pip`, `setuptools` and `wheel` values.